views/logIn.hbs

```diff
@@ -0,0 +1,18 @@
+<form action="/login" method="POST" id="form">
+  <h2>Log in</h2>
+  <label for="username">Username</label>
+  <input type="text" name="username" required="true">
+  <label for="password">Password</label>
+  {{!-- Nota**: No es suficiente con bloquearlo desde el Front --}}
+  <input type="password" name="password" required="true">
+
+  {{#if errorMessage }}
+    <div>{{errorMessage}}</div>
+  {{/if}}
+
+  <button><a href="/">Sign in</a></button>
```

Comment:
Ojo! El botón no lleva enlace. Es de tipo 'Submit' (envía el formulario) y la que renderiza una vista u otra es la ruta contra la que se envía el formulario POST. En este caso la ruta **/login**

`<button type="submit">Sign in</button>`